catalog/Podfile

```diff
@@ -4,7 +4,7 @@ project 'MDCCatalog.xcodeproj'
 target "MDCCatalog" do
   project 'MDCCatalog.xcodeproj'
   pod 'MaterialComponentsCatalog', :path => '../'
-  pod 'MaterialComponents', :path => '../'
+  pod 'MaterialComponents'
```

Comment:
i think we still want to pull locally so that we can test local things. This will make it so the catalog will pull from a tagged release?